pkg/asset/installconfig/azure/validation.go

```diff
@@ -128,19 +129,20 @@ func ValidateInstanceType(client API, fieldPath *field.Path, region, instanceTyp
 			}
 		} else if strings.EqualFold(*capability.Name, "UltraSSDAvailable") {
 			ultraSSDAvailable = strings.EqualFold(*capability.Value, "True")
-		} else if strings.EqualFold(*capability.Name, string(aztypes.VMnetworkingTypeAccelerated)) {
-			if vmNetworkingType == string(aztypes.VMnetworkingTypeAccelerated) && !strings.EqualFold(*capability.Value, "True") {
-				errMsg := fmt.Sprintf("vm networking type is not supported for instance type %s", instanceType)
-				allErrs = append(allErrs, field.Invalid(fieldPath.Child("vmNetworkingType"), vmNetworkingType, errMsg))
-			}
+		} else if strings.EqualFold(*capability.Name, "AcceleratedNetworkingEnabled") {
```

Comment:
I created a type for this magic string here:
https://github.com/openshift/installer/blob/master/pkg/types/azure/machinepool.go#L50

Reply:
Ah! I switched to the type in the pr.